tests/Doctrine/Tests/ORM/EntityManagerTest.php

```diff
@@ -288,4 +291,34 @@ public function testGetReferenceRetrievesReferencesWithGivenProxiesAsIdentifiers
         self::assertTrue($this->em->contains($simpleIdReferenceB));
         self::assertTrue($this->em->contains($nestedIdReference));
     }
+
+    public function testCallingFlushInsideFlush()
+    {
+        $listener = new class($this)
+        {
+            /** @var OrmTestCase */
+            private $testCase;
+
+            /** @var bool This flag is used to prevent infinity loop in case of regression */
```

Comment:
An infinite loop in case of a regression here is OK, since it would lead to an infinite stack trace, which (with xDebug enabled) would simply lead to a crash.

Reply:
So all of this can be dropped :+1: